ui/StatusQ/src/StatusQ/Components/StatusChatList.qml

```diff
@@ -35,10 +35,11 @@ Item {
     StatusListView {
         id: statusChatListItems
         width: 288
-        height: contentHeight
+        height: root.height
```

Comment:
how is this different than `height: contentHeight`?

Reply:
the implicit size of the StatusChatList is the size of the listview (no scroll), but now we can use StatusChatList as a normal listview also (having scroll) by providing the size to StatusChatList.

`height: contentHeight` will set the listview size to contentHeight with no option to update the ListView height
`height: root.height` will allow ListView resizing by setting StatusChatList.height from outside